plugin-name/plugin-name.php

```diff
@@ -31,7 +31,7 @@
 }
 
 /**
- * Currently plugin version.
+ * Currentl plugin version.
```

Comment:
I think you've fixed a typo by replacing it with a new typo. :stuck_out_tongue: It should probably be "Current".